utils/src/field_helpers.rs

```diff
@@ -122,26 +122,27 @@ pub trait FieldHelpers<F> {
     where
         F: PrimeField,
     {
-        F::size_in_bits() / 8 + (F::size_in_bits() % 8 != 0) as usize
+        (F::MODULUS_BIT_SIZE / 8) as usize + (F::MODULUS_BIT_SIZE % 8 != 0) as usize
     }
 
     /// Get the modulus as `BigUint`
     fn modulus_biguint() -> BigUint
     where
         F: PrimeField,
     {
-        BigUint::from_bytes_le(&F::Params::MODULUS.to_bytes_le())
+        BigUint::from_bytes_le(&F::MODULUS.to_bytes_le())
     }
 }
 
 impl<F: Field> FieldHelpers<F> for F {
     fn from_bytes(bytes: &[u8]) -> Result<F> {
-        F::deserialize(&mut &*bytes).map_err(|_| FieldHelpersError::DeserializeBytes)
+        F::deserialize_uncompressed(&mut &*bytes).map_err(|_| FieldHelpersError::DeserializeBytes)
```

Comment:
In arkworks.0.3.0, we have:
```rust
pub trait CanonicalDeserialize: Sized {
    /// Reads `Self` from `reader`.
    fn deserialize<R: Read>(reader: R) -> Result<Self, SerializationError>;

    /// Reads `Self` from `reader` without compression.
    #[inline]
    fn deserialize_uncompressed<R: Read>(reader: R) -> Result<Self, SerializationError> {
        Self::deserialize(reader)
    }

    /// Reads `self` from `reader` without compression, and without performing
    /// validity checks. Should be used *only* when the input is trusted.
    #[inline]
    fn deserialize_unchecked<R: Read>(reader: R) -> Result<Self, SerializationError> {
        Self::deserialize_uncompressed(reader)
    }
}
```
Then, `deserialize` is exactly `deserialize_uncompressed`. it seems to perform the check at the same time, based on the documentation.
In arkworks 0.4.2, we have
```rust
pub trait CanonicalDeserialize: Valid {
    /// The general deserialize method that takes in customization flags.
    fn deserialize_with_mode<R: Read>(
        reader: R,
        compress: Compress,
        validate: Validate,
    ) -> Result<Self, SerializationError>;

    fn deserialize_compressed<R: Read>(reader: R) -> Result<Self, SerializationError> {
        Self::deserialize_with_mode(reader, Compress::Yes, Validate::Yes)
    }

    fn deserialize_compressed_unchecked<R: Read>(reader: R) -> Result<Self, SerializationError> {
        Self::deserialize_with_mode(reader, Compress::Yes, Validate::No)
    }

    fn deserialize_uncompressed<R: Read>(reader: R) -> Result<Self, SerializationError> {
        Self::deserialize_with_mode(reader, Compress::No, Validate::Yes)
    }

    fn deserialize_uncompressed_unchecked<R: Read>(reader: R) -> Result<Self, SerializationError> {
        Self::deserialize_with_mode(reader, Compress::No, Validate::No)
    }
}
```
Which seems to be the expected behavior, as we would have called `deserialize_uncompressed_unchecked` if we did not want to validate.